platform-includes/performance/improving-data/unity.mdx

```diff
@@ -0,0 +1,47 @@
+## Improving Data on Transactions and Spans
+
+### Adding Data Attributes to Transactions
+
+You can add data attributes to your transactions. This data is visible in the trace explorer in Sentry. Data attributes can be of type `string`, `number` or `boolean`, as well as arrays of these types:
+
+```csharp
+ITransaction transaction = SentrySdk.StartTransaction("processOrderBatch()", "task");
+transaction.SetData("my-data-attribute-1", "value1");
+transaction.SetData("my-data-attribute-2", 42);
+transaction.SetData("my-data-attribute-3", true);
+
+transaction.SetData("my-data-attribute-4", new[] {"value1", "value2", "value3"});
+transaction.SetData("my-data-attribute-5", new[] {42, 43, 44});
+transaction.SetData("my-data-attribute-6", new[] {true, false, true});
+```
+
+<Alert level="info">
+The <code>SetExtra</code> method is still supported but has been deprecated in favor of <code>SetData</code>.
+</Alert>
```

Comment:
Code itself already does the job of warning and since we didn't document SetExtra for thise use case before I think it's fine to remove the note:

https://github.com/getsentry/sentry-dotnet/blob/b1d397211d0070665725ea003f1c4356b1748ff9/src/Sentry/SentryTransaction.cs#L342-L344

```suggestion
```